Netwrix/netwrix_auditor/ingest/parser.yml

```diff
@@ -0,0 +1,90 @@
+name: netwrix_auditor_parser
+pipeline:
+  - name: cef_parser
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general-setter
+    description: ''
+    filter: ''
+    external: null
+  - name: user-email-setter
+    description: ''
+    filter: '{{ "@" in cef_parser.message.suser }}'
+    external: null
+  - name: user-name-domain-setter
+    description: ''
+    filter: '{{ "\\" in cef_parser.message.suser }}'
+    external: null
+  - name: user-only-setter
+    description: ''
+    filter: '{{ "\\" not in cef_parser.message.suser and "@" not in cef_parser.message.suser }}'
+  - name: document-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.cat == "Document" }}'
+    external: null
+  - name: logon-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Logon Activity" }}'
+    external: null
+  - name: exchange-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Exchange Online" }}'
+    external: null
+stages:
+  general-setter:
+    actions:
+      - set:
+          event.kind: 'event'
+          event.code: '{{ cef_parser.message.DeviceEventClassID |lower }}'
+          event.severity: '{{ cef_parser.message.Severity }}'
+          event.start: '{{ cef_parser.message.start }}'
```

Comment:
I think you should use the stage date to parse this date in the message as [`event.start`](https://www.elastic.co/guide/en/ecs/8.4/ecs-event.html#field-event-start) accept date type (ie [RFC3339](https://www.rfc-editor.org/rfc/rfc3339#page-10) formatted date)